src/main/java/com/exasol/ReportWriter.java

```diff
@@ -0,0 +1,87 @@
+package com.exasol;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import com.exasol.validation.ValidationReport;
+
+/**
+ * This class writes reports from {@link ValidationReport} to a file.
+ */
+public class ReportWriter {
+    private static final Logger LOGGER = Logger.getLogger(ReportWriter.class.getName());
+    private static final String RELEASE_ROBOT_REPORT = "/.release-robot/last_report.txt";
```

Comment:
I'd be happy to add one but I don't have integration tests at all :D GitHub doesn't provide a testing environment, maven central is even worse, so we live with unit tests

Reply:
Added a FILE_SEPARATOR constant

Reply:
Why not Path.of()? It's natie ajav and much more readable.

Reply:
Because HOME_DIRECTORY is a String, and I need to concatenate it with the RELEASE_ROBOT_REPORT. And the File class takes a String, not a path. So it doesn't really make sense to use Path here if I need to convert it from a string back to a string

Reply:
use [`Path#toFile()`](https://www.geeksforgeeks.org/path-tofile-method-in-java-with-examples/)